### DISCUSSION TITLE
How to render an image? | Como renderizar uma imagem?

### DISCUSSION THREAD

**Discussion by JoaoAlavarse** [wppconnect-team/wppconnect#2410] [Q&A]
I'm trying to render an image with the base64 body, but the quality is really bad, what can i do to show this image correctly?

Estou tentando renderizar uma imagem com o conteúdo do body em base64, mas a qualidade fica muito ruim e não da para visualizar a imagem, o que posso fazer?

**Comment by manfe** [comment]
@JoaoAlavarse the base64 that comes with an image it's just a thumbnail, you need to download the media to be able to show it properly.
